### DISCUSSION TITLE
Calculate_all() problem

### DISCUSSION THREAD

**Discussion by desoka** [Dlubal-Software/RSTAB_Python_Client#36] [Q&A]
I try to read out the results of a certain load case via:
sgr_lc1 = ResultTables.MembersInternalForces(CaseObjectType.E_OBJECT_TYPE_LOAD_CASE, 1)

Though I receive an error saying no results available (suds.WebFault: b"Server raised fault:).
Is Calculate_all() missing any parameters?

Thanks.

**Comment by heetrojivadiya** [comment]
Hello @desoka, it seems like you might forget to initiate calculation properly before extracting results. For better investigation, you should provide code or detailed snapshot.
